srcpkgs/fcft/template

```diff
@@ -1,17 +1,22 @@
 # Template file for 'fcft'
 pkgname=fcft
-version=2.2.7
+version=2.3.1
 revision=1
 wrksrc=$pkgname
 build_style=meson
+configure_args="-Dtext-shaping=$(vopt_if harfbuzz enabled disabled)"
 hostmakedepends="pkg-config scdoc"
-makedepends="fontconfig-devel freetype-devel pixman-devel tllist"
+makedepends="fontconfig-devel freetype-devel pixman-devel tllist
+ $(vopt_if harfbuzz harfbuzz-devel)"
 short_desc="Simple library for font loading and glyph rasterization"
 maintainer="Isaac Freund <ifreund@ifreund.xyz>"
 license="MIT"
 homepage="https://codeberg.org/dnkl/fcft"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=d1c6f540bb75a3b66705ff72d2445b2e084cd427ffa62cbe79b7f44047773e96
+checksum=4a8ed7e9818a391ec422ec8ee561fccf2586410c1bfc86ecc64026a5cf8bc18c
+
+build_options="harfbuzz"
+build_options_default="harfbuzz"
```

Comment:
Do you want to add a desc_option, too?

Reply:
We already have one in `common/options.description` :)